src/main/java/org/marketdesignresearch/mechlib/outcomerules/ccg/blockingallocation/XORBlockingCoalitionFinderFactory.java

```diff
@@ -18,7 +18,7 @@ public XORBlockingCoalitionFinderFactory(BigDecimal epsilon) {
     }
 
     public XORBlockingCoalitionFinderFactory() {
-        this.epsilon = PrecisionUtils.EPSILON;
+        this.epsilon = PrecisionUtils.EPSILON.scaleByPowerOfTen(2);
```

Comment:
I had an issue in GSVM that raised an exception here. I think it is because the lower bound was too tight. Therefore I added some more slack here. The slack is only used to compute a lower bound on the objective value. In a max problem this does not change the optimal solution, but if the lower bound is too tight (numerical issues with sequential MIP calls) the problem might get infeasible.

Reply:
That's fine with me - can you reproduce it such that you know that this solves the issue in the situation you encounter?

Reply:
I ran large number of instances and therfore reproducing an error is quite some work. But I'm quite sure that this resolve the issue. Anyway, the overall problem stays the same and therefore this adjustment is safe in any case.